src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,119 @@
+package duke.ui;
+import duke.logic.TaskList;
```

Comment:
I notice other files have the line to separate package and import statements except for this. Would be good for readability to standardize according to src/main/java/duke/logic/Storage.java

Reply:
It could be the files that were written earlier that result in the inconsistent ordering of import statements.